src/action-creators/deleteEmptyThought.js

```diff
@@ -21,6 +21,7 @@ import {
   rootedContextOf,
   splitChain,
 } from '../util.js'
+import { getNextRank } from '../util/getNextRank.js'
```

Comment:
Please import this in the `util` import block above.

Reply:
Sure! Never rely on auto import 😛

Reply:
Ooh, didn't know that was a thing! Not in Sublime world at least.